### PR TITLE
Lint update - Jan 2019, TESS Light Curve Viewer

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Overview
Package: `lib-classifier`

We had some code that needed lintin', so lintin' we did.
- Ran `npm run lint`
- ~~Running~~ Ran some manual checks for the TESS Light Curve Viewer

### Status
~~Gimme a sec to finish my manual checks.~~

OK, ready.

### CODE REVIEW

**Comment by shaunanoordin** [comment]
I've done a quick lint of the `lib-classifier` code, with a focus on cleaning my TESS LCV additions, but I'm thinking there's always more that can be cleaned up in a later sweep.

Although there _is_ one more additional thing I'd like to fix in this PR if anyone has any idea how to address it:

```
// Lint output
front-end-monorepo/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/helpers/getViewer/getViewer.js
  1:21  error  Use default import syntax to import 'SingleImageViewer'
  2:21  error  Use default import syntax to import 'LightCurveViewer'

// Code in getViewer.js
import { default as SingleImageViewer } from '../../components/SingleImageViewer'
import { default as LightCurveViewer } from '../../components/LightCurveViewer'
``` 

🤷‍♂️

**Comment by rogerhutchings** [comment]
```
import SingleImageViewer from '../../components/SingleImageViewer'
import LightCurveViewer from '../../components/LightCurveViewer'
```

...or am I missing something?

**Comment by shaunanoordin** [comment]
I thought that was the obvious solution too, but that broke the Classifier
when I tried. :shrug: I'll give this another shot on Monday, I probably
derped on something simple.

File: packages/lib-classifier/src/components/Classifier/components/MetaTools/components/MetadataModal/MetadataModal.js [hunk 1]
```diff
@@ -39,13 +39,13 @@ export default function MetadataModal (props) {
...
-      property: 'value',
-      header: <SpacedText>{counterpart('MetadataModal.table.value')}</SpacedText>,
-    }
+    property: 'label',
+    header: <SpacedText>{counterpart('MetadataModal.table.label')}</SpacedText>,
+    primary: true
+  }, {
+    property: 'value',
+    header: <SpacedText>{counterpart('MetadataModal.table.value')}</SpacedText>
+  }
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/MetaTools/components/MetadataModal/MetadataModal.js]: ugh I hate this indentation.

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskInputField/TaskInputField.js [hunk 2]
```diff
@@ -101,9 +101,9 @@ export const StyledTaskInputField = styled.label`
     border-width: 2px;
     border-style: solid;
     border-color: ${theme('mode', {
-      dark: zooTheme.global.colors.darkTeal,
-      light: zooTheme.global.colors.teal
-    })};
+    dark: zooTheme.global.colors.darkTeal,
+    light: zooTheme.global.colors.teal
+  })};
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskInputField/TaskInputField.js]: boo on this indentation

**Pull request merged**